contrib/kube-prometheus/assets/prometheus/rules/general.rules

```diff
@@ -0,0 +1,109 @@
+### Up Alerting ###
+
+Alert TargetDown
+  IF up == 0
+  FOR 10m
+  LABELS {
+    severity = "warning"
+  }
+  ANNOTATIONS {
+    summary = "target is down",
+    description = "A target of type {{ $labels.job }} is down."
+  }
+
+### Dead man's switch ###
+
+ALERT DeadMansSwitch
+  IF vector(1)
+  LABELS {
+    severity = "none",
+  }
+  ANNOTATIONS {
+    summary = "Alerting DeadMansSwitch",
+    description = "This is a DeadMansSwitch meant to ensure that the entire Alerting pipeline is functional.",
+  }
+
+### File descriptor alerts ###
+
+ALERT TooManyOpenFiles
+  IF 100*process_open_fds / process_max_fds > 50
+  FOR 10m
+  LABELS {
+    severity = "warning"
+  }
+  ANNOTATIONS {
+    summary = "too many open file descriptors",
+    description = "{{ $labels.job }}: {{ $labels.namespace }}/{{ $labels.pod }} is using {{ $value }}% of the available file/socket descriptors.",
+  }
+
+ALERT K8STooManyOpenFiles
+  IF 100*process_open_fds / process_max_fds > 80
```

Comment:
Also, this being a capacity alert, it should likely be using `predict_linear`. I see those are right below. Maybe just drop these here then or replace them with a single one at 95%? The alert name is questionable too as it has nothing to do with K8S and also doesn't fix it to a k8s job.